api/management/commands/acquiredata.py

```diff
@@ -0,0 +1,10 @@
+import os
+from urllib.request import urlretrieve
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        path = os.path.join('data', 'diseases.tsv')
+        url = 'https://raw.githubusercontent.com/cognoma/cancer-data/master/download/diseases.tsv'
```

Comment:
I think this would be an excellent feature, but it seemed nontrivial to implement. For instance, the urls in old commits had the full SHA value in them, so our user might need to provide this. If you see an easy/clever way to do it, I would be up to give it a try.